string name
bool is_group_name
---
